pandas/formats/excel.py

```diff
@@ -0,0 +1,586 @@
+"""Utilities for conversion to writer-agnostic Excel representation
+"""
+
+import re
+import warnings
+import itertools
+
+import numpy as np
+
+from pandas.compat import reduce
+from pandas.formats.css import CSSResolver, CSSWarning
+from pandas.formats.printing import pprint_thing
+from pandas.types.common import (is_float)
+import pandas._libs.lib as lib
+from pandas.core.index import Index, MultiIndex
```

Comment:
yes because its imported in ``pandas/core/frame.py``, which we don't want to do (this creates the circular imports). When you change it in frame.py this will just work.